packages/nocodb/src/controllers/users/users.controller.ts

```diff
@@ -136,22 +84,19 @@ export class UsersController {
   @Post(`/auth/google/genTokenByCode`)
   @HttpCode(200)
   @UseGuards(AuthGuard('google'))
-  async googleSignin(@Request() req) {
-    return this.usersService.login(req.user);
+  async googleSignin(@Request() req, @Response() res) {
+    await this.setRefreshToken({ req, res });
+    res.json(this.usersService.login(req.user));
```

Comment:
Possible to cause similar issue, see other comment.

Reply:
That method is synchronous so I removed `async` from the method